Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -23,6 +24,13 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
-    // Add your necessary variables here
+    double totalAmpHoursUsed_;
+    double averageCurrent_; //
+    double totalCurrent_;
+    int counter_;
+    double hours_remaining_;
+    double current_;
+    int initialAhUsed_;
+    QTime initialTime_;
+    QTime previousMilliseconds_;
```

Comment:
I thought it does contain the milliseconds of the previous time, the way I have it set up.